COBOL Programming Course #2 - Advanced Topics/COBOL Programming Course #2 - Advanced Topics.md

```diff
@@ -549,7 +549,325 @@ The result is execution of COBOL program CBLDB21 to read the Db2 table and write
 7. Two additional COBOL programs with Db2 API exist, CBLDB22 and CBLDB23 using the same Db2 table as the data source.
 
 
+
+\newpage
+
+# SORT and MERGE
```

Comment:
Please review how other chapters are set up with a brief introduction followed by a table of contents for the chapter.